frontend/beCompliant/src/components/table/LastUpdated.tsx

```diff
@@ -2,10 +2,12 @@ import { Stack, Text } from '@kvib/react';
 import { formatDateTime } from '../../utils/formatTime';
 
 type Props = {
-  updated: Date;
+  updated?: Date;
 };
 
 export function LastUpdated({ updated }: Props) {
+  if (!updated) return <></>;
```

Comment:
```suggestion
  if (!updated) return null;
```

Dette er kanskje mer idiomatisk